### PR TITLE
Add CHANGELOG

### PR DESCRIPTION
**Pull request by itamargiv**
As per discussion on #265, adding a changelog file to keep track of changes prior ro making release notes.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# WiKit Changelog
+
+_**Important:** This file contains a draft to the official release notes, and thus is not a stable documentation of the upcoming version (as it will frequently change during development)_
+
+## v2.0.0 (development)
+
+**Breaking Changes:**
+- Rename `Button` component's `type` property to `flavor`
```
Comment (SaiSan-WMDE) [inline CHANGELOG.md:8]:
```suggestion
- Remove widths properties #259 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- Added Message component
+- First version of Lookup component
+- Exports type declaration files #223
+- More Icons for the Icon component
+
+**Bug Fixes:**
+- Re-enabled type checking
+
+**Miscellaneous:**
+- Started CHAGELOG file
```
Comment (micgro42) [inline CHANGELOG.md:20]:
```suggestion
- Started CHANGELOG file
```

File: docs/src/00-doc/04_release_management.stories.mdx [hunk 3]
```diff
@@ -28,10 +28,14 @@ It is also their responsibility to inform other relevant roles like UX and PM (i
 
 Once your changes are merged and you want to release a new version of the components library you need to take the following steps:
  - To release a new version run the command `npm run bump-version -- [major | minor | patch]`. This will take care of updating the version number in all package.json files.
+ - Proofread and review the `CHANGELOG.md` file to ensure all changes are included (this will be later used in the release notes).
+ - Add a new heading to the chagelog with the next expected version number and mark it as the devlopment section
```
Comment (micgro42) [inline docs/src/00-doc/04_release_management.stories.mdx:32]:
```suggestion
 - Add a new heading to the chagelog with the next expected version number and mark it as the development section 
```

**Review by micgro42 — DISMISSED** [review]
Looks good, I just found two typos

**Review by tzhelyazkova — COMMENTED** [review]
Apparently there isn't any universal standard for change logs but this website https://keepachangelog.com/en/1.0.0/ aims to collect good practices and eventually become a standard. Shall we follow that?
One main difference I see is that upcoming changes are collected under an "Unreleased" section.
Also, they list all possible types of changes one can have which is a fantastic thing imo.

**Comment by itamargiv** [comment]
> Shall we follow that?

I like this :heart:, yes i think we should follow this standard. The only addition I would make is to probably flag breaking changes more clearly somehow (but that can be easily done with styling and a parentheses comment). Will change.

**Review by tzhelyazkova — APPROVED** [review]
Nice!

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**